fms_sdg/databuilders/simple/generate.py

```diff
@@ -32,7 +32,7 @@ def __init__(
         self,
         *args: Any,
         num_prompt_instructions: Optional[int] = 2,
```

Comment:
As we have valid default, why not?
```suggestion
        num_prompt_instructions: int = 2,
```